### ISSUE TITLE
Slight revision to install guide needed

### ISSUE THREAD

**Issue by tofr** [jay0lee/GAM#811]
Due to ubuntu 18.04.1 somehow deleting my GAM install, I was forced to re-install just now...

This part of the install guide needs a slight revision:
-------------------------------------------
Now go back to your browser and copy your client secret.
Enter your Client Secret: dIAAHzOjVFZqoGBNw5TazU8U
Almost there! Now please switch back to your browser and:

1. Click OK to close "OAuth client" popup if it's still open.
2. Click "Manage service accounts" on the right of the screen.
3. Click the 3 dots to the right of your service account.
4. Choose Edit.
5. Check the "Enable G Suite Domain-wide Delegation" box and click Save.
-------------------------------------------

===================================
Step 3 should read something like this, as the 3 dots referenced do not seem present any more.

3. Click hamburger in upper left corner, then "IAM & admin", then "Service accounts.

**Comment by jay0lee** [comment]
Hint: anyone can edit the wiki...

**Comment by johnkershaw** [comment]
'Mr Anyone' here :)

So I went to the install page to make this quick-fix change, and decided I might as well single step through the instructions whilst I was there to check they're still current.

Using Google Shell, I got as far as [clicking the link to to initialise the process](https://console.developers.google.com/flows/enableapi?apiid=admin,appsactivity,calendar,classroom,drive,gmail,groupssettings,licensing,plus,contacts) but after clicking 'Continue' (with 'Create a project' selected) it bombs out with a 'Undefined parameters' apiNames:

![screenshot 2018-12-04 at 7 49 33 pm](https://user-images.githubusercontent.com/5265410/49443501-343e0100-f7ff-11e8-8c4c-a88fc2d546e7.png)

I tried it 3 times, same result.

Has something changed at Google's end causing this to no longer work as expected? Or am I missing something obvious?
